data/saas/config/alchemer_config.yml

```diff
@@ -0,0 +1,95 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Alchemer
+  type: alchemer
+  description: A sample schema representing the Alchemer erasure only integration for Fides
+  version: 0.1.0
+
+  connector_params:
+    - name: domain
+      default_value: api.alchemer.com
+    - name: api_token
+      default_value: <api_key>
+    - name: api_token_secret
+      default_value: <api_key_secret>
+
+  client_config:
+    protocol: https
+    host: <domain>
+    authentication:
+      strategy: no_authentication
```

Comment:
It is the case that we would want to remove this part since it's basically 'nothing'?